src/logging_unittest.cc

```diff
@@ -995,6 +997,86 @@ static void TestCustomLoggerDeletionOnShutdown() {
   EXPECT_FALSE(IsGoogleLoggingInitialized());
 }
 
+namespace LogTimes {
+// Log a "message" every 10ms, 10 times. These numbers are nice compromise
+// between total running time of 100ms and the period of 10ms. The period is
+// large enough such that any CPU and OS scheduling variation shouldn't affect
+// the results from the ideal case by more than 1% (100us or 0.1ms)
+static GLOG_CONSTEXPR double LOG_PERIOD_SEC = 0.01;
```

Comment:
No `static` needed for `constexpr` or `const`  variables.

Reply:
Unless I want internal linkage, right? In this case I only want these accessible inside the unit test.

Reply:
`constexpr` implies `const` which in turn implies internal linkage.